packages/zero-cache/src/db/create.ts

```diff
@@ -1,18 +1,15 @@
+import {assertValidLiteColumnSpec} from '../types/lite.ts';
 import {id, idList} from '../types/sql.ts';
 import type {ColumnSpec, LiteIndexSpec, LiteTableSpec} from './specs.ts';
 
+/**
+ * `spec` always comes from `mapPostgresToLiteColumn` so we only need to support
```

Comment:
so there will be not be string coming through to here without `TEXT_ARRAY`, even if someone ran a canary from a commit after we removed `TEXT_ARRAY`?

Reply:
no. I traced this. All the specs com from this function.